src/main.ts

```diff
@@ -69,11 +70,11 @@ async function run(): Promise<void> {
     // and then made it as terse as I could. :shrug:
 
     const g = octokit.rest.git;
-    const {data: {object: {sha: commit_sha}}} = await g.getRef({owner, repo, ref});
+    const {data: {object: {sha: commit_sha}}} = await g.getRef({owner, repo, ref: `heads/${ref}`});
```

Comment:
Nit: Would it be worth renaming `ref` to `sha` or `branch` and then making `ref=\`heads/${sha}`.

Main rationale is that it would remove the risk of someone using ref as is in the future.

Reply:
Good call. Done in 3c367624be763e3ed333a439b49cf3358a1c9c16.